test/plots/london-maps.ts

```diff
@@ -0,0 +1,72 @@
+import * as Plot from "@observablehq/plot";
+import * as d3 from "d3";
+import {feature} from "topojson-client";
+
+export async function londonMap() {
+  const london = feature(await d3.json("data/london.json"), "boroughs");
+  return Plot.plot({
+    projection: {type: "transverse-mercator", rotate: [2, 0, 0], domain: london},
+    marks: [
+      Plot.geo(london),
+      Plot.text(london.features, Plot.centroid({text: "id", stroke: "var(--plot-background)", fill: "currentColor"}))
+    ]
+  });
+}
+
+export async function londonCarAccess() {
+  const london = feature(await d3.json("data/london.json"), "boroughs");
+  const wide = await d3.csv<any>("data/london-car-access.csv", d3.autoType);
+  const access = wide.flatMap(({borough, y2001, y2011, y2021}) => [
+    {borough, year: "2001", access: y2001},
+    {borough, year: "2011", access: y2011},
+    {borough, year: "2021", access: y2021}
+  ]);
+  const boroughs = new Map(london.features.map((d) => [d.id, d]));
+  return Plot.plot({
+    width: 900,
+    projection: {type: "transverse-mercator", rotate: [2, 0, 0], domain: london},
+    color: {scheme: "RdYlBu", pivot: 0.5},
+    marks: [
+      Plot.geo(
+        access,
+        Plot.centroid({
+          fx: "year",
+          geometry: (d) => boroughs.get(d.borough),
+          fill: "access",
+          stroke: "var(--plot-background)",
+          strokeWidth: 0.75,
+          tip: true
+        })
+      )
```

Comment:
I guess I’m not sure we should/need to support this use case now. A more explicit way to write this is:

```suggestion
      Plot.geo(access, {
        fx: "year",
        geometry: (d) => boroughs.get(d.borough),
        fill: "access",
        stroke: "var(--plot-background)",
        strokeWidth: 0.75
      }),
      Plot.tip(
        access,
        Plot.pointer(Plot.centroid({fx: "year", geometry: (d) => boroughs.get(d.borough), fill: "access"}))
      )
```

I don’t think it’s right to use the centroid/geoCentroid transform with the geo mark if the geo mark doesn’t need the _x_ and _y_ channels — if they’re just intended to be used with the **tip** option. It feels like we should just make the **tip** option work directly with the geo mark rather than going halfway.